segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageTests.java

```diff
@@ -719,6 +719,38 @@ public void testWrite() throws Exception {
         checkDataRead(testSegmentName, testContext, 0, total);
     }
 
+
+    /**
+     * Test Write after repeated failure.
+     *
+     * @throws Exception Exception if any.
+     */
+    @Test
+    public void testWriteAfterWriteFailure() throws Exception {
+        String testSegmentName = "foo";
+        TestContext testContext = getTestContext();
```

Comment:
Just like in the other PR, please add cleanup methods to this class.

Reply:
added